recipes/github-contributes-heatmap/demos/commit-activity-demo/index.html

```diff
@@ -0,0 +1,580 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <title>Базовый вариант — GitHub-плитка — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap">
+  <style>
+    html {
+      color-scheme: dark;
+    }
+
+    body {
+      min-height: 100vh;
+      margin: 0;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      box-sizing: border-box;
+      background-color: #18191C;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+    }
+
+    .main-container {
+      display: flex;
+      flex-direction: column;
+      gap: .5rem;
+      min-width: 200px;
+      padding: 1rem;
+      position: relative;
+
+      --base-color: hsl(112 100% 50%);
+      --text-color: #FFFFFF;
+      --label-color: #C56FFF;
+    }
+
+    .main-container div {
+      box-sizing: border-box;
+    }
+
+    .description {
+      display: flex;
+      flex-direction: row;
+      gap: .5rem;
+      flex-wrap: wrap;
+      font-size: 1.5rem;
```

Comment:
```suggestion
      font-size: 1rem;
```

Давай здесь, пожалуй, не будем шрифт увеличивать (это ведь не заголовок) и отдадим всю сцену офигеннейшей плитке 💚 И в тыквенной демке тоже, конечно 🧡

(Нет времени объяснять, нужно ставить сердечки всех использованных цветов 💜)